src/defer-load.directive.ts

```diff
@@ -20,19 +21,21 @@ export class DeferLoadDirective implements AfterViewInit, OnDestroy {
         @Inject(PLATFORM_ID) private platformId: Object
     ) { }
 
-    public ngAfterViewInit () {
-        if (isPlatformBrowser(this.platformId)) {
+    public ngOnInit () {
+        if (isPlatformServer(this.platformId) && this.preRender === true) {
+            this.load();
+        } else {
             if (this.hasCompatibleBrowser()) {
                 this.registerIntersectionObserver();
                 if (this._intersectionObserver && this._element.nativeElement) {
                     this._intersectionObserver.observe(<Element>(this._element.nativeElement));
                 }
             } else {
-                this.addScrollListeners();
-            }
-        } else {
-            if (this.preRender === true) {
-                this.load();
+                if (this.fallbackEnabled) {
```

Comment:
this should be a proper boolean check, otherwise you will accidentally enable the fallback when the attribute value is a string value

`fallbackEnabled="false"` vs `[fallbackEnabled]="false"`